book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd

```diff
@@ -99,22 +99,24 @@ This will not go into much detail about about how to use Docker -- for more deta
     b. "Initialize" the data for the PEcAn database.
 
         ```bash
-        docker-compose run --rm bety initialize
+        docker run --rm --network pecan_pecan pecan/db
         ```
         
         This should produce a lot of output describing the database operations happening under the hood.
-        Some of these will look like errors (including starting with `ERROR`), but _this is normal_.
```

Comment:
Is this line removed because no error-like messages are expected anymore, or for some other reason? This line has reassured me on several occasions and I'd vote to keep it unless the expected output is now truly warning-free.

Reply:
there are still error messages but not as obvious as before and they don't start with `ERROR` anymore, maybe we can just delete the parenthesis bit (because it could also be confusing to not see messages starting with `ERROR` at first glance)
```

docker run --rm --network pecan_pecan pecan/db

Unable to find image 'pecan/db:latest' locally
latest: Pulling from pecan/db
cbdbe7a5bc2a: Pull complete 
0826c3db658a: Pull complete 
1e6696f14f8a: Pull complete 
Digest: sha256:5cb1c84ac2afaa3e7e4f78fbd6e55d5d24822cea6b0e29294df3014173c41212
Status: Downloaded newer image for pecan/db:latest
CREATE ROLE
pg_restore: connecting to database for restore
pg_restore: while INITIALIZING:
pg_restore: error: could not execute query: ERROR:  unrecognized configuration parameter "idle_in_transaction_session_timeout"
Command was: SET idle_in_transaction_session_timeout = 0;
pg_restore: dropping DATABASE bety
pg_restore: while PROCESSING TOC:
pg_restore: from TOC entry 4783; 1262 20600 DATABASE bety bety
pg_restore: error: could not execute query: **ERROR**:  database "bety" does not exist
Command was: DROP DATABASE bety;
pg_restore: creating DATABASE "bety"
pg_restore: connecting to new database "bety"
pg_restore: connecting to database "bety" as user "postgres"
pg_restore: error: could not execute query: **ERROR**:  unrecognized configuration parameter "idle_in_transaction_session_timeout"
Command was: SET idle_in_transaction_session_timeout = 0;
pg_restore: creating SCHEMA "admin"
pg_restore: creating SCHEMA "public"
pg_restore: from TOC entry 8; 2615 22127 SCHEMA public bety
pg_restore: error: could not execute query: **ERROR**:  schema "public" already exists
Command was: CREATE SCHEMA public;
```

Reply:
Ok....so I am adding a line that says: "Some of these will look like errors, but _this is normal_.". I think this will address the fact that some statements actually do have the word "error" in them, but also that they do not start with the word "error"